### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/strings.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#181]
```
italian revision, completed

Giulio
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 11 Apr 2011 at 8:00

Attachments:
- [strings.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-181/comment-0/strings.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thank you again liogiu!

Committed as part of r70.
```

Original comment by `oskar.wi...@gmail.com` on 14 Apr 2011 at 8:34
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:40
- Added labels: **Type-Translation**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:51
- Changed state: **Done**
